### PR TITLE
[Database metrics] Functions for parsing metrics

### PR DESCRIPTION
**Pull request by chinandrew**
Summary of changes:
- Add functions that take in the raw outputs from various metric capturing functions and parse them into a more usable data type.

### CODE REVIEW

File: tests/database_metrics/test_parsers.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+"""Tests for parsers.py"""
```
Comment (dfarrow0) [inline tests/database_metrics/test_parsers.py:1]: note that this repo uses python's built-in unittest (through a wrapper called py3tester), which is not the testing framework used by the indicators repo. so i don't think this is going to pass, which will cause auto-deploy to fail.

**Review by dfarrow0 — CHANGES_REQUESTED** [review]

File: tests/database_metrics/test_parsers.py (hunk 1 above)
Comment (dfarrow0) [reply tests/database_metrics/test_parsers.py:1]: some recent unit tests as a sample here: https://github.com/cmu-delphi/operations/blob/main/tests/screenshots/test_utils.py

docs on running unit tests: https://github.com/cmu-delphi/operations/blob/main/tests/screenshots/test_utils.py

tldr:

```
docker run --rm delphi_python \
  python3 -m undefx.py3tester.py3tester --color repos/delphi/operations/tests
```

Comment (chinandrew) [reply tests/database_metrics/test_parsers.py:1]: Thanks for calling this out, didn't realize that was the case. I'll follow the same pattern as the other tests and ping you once that's done.

Comment (dfarrow0) [reply tests/database_metrics/test_parsers.py:1]: no problem. as far as i know, indicators is the only delphi python code thats _isn't_ tested with unittest.

(marginally related side note: indicators is also the only delphi python code that's indented with 4 vs 2 spaces hah.. but not a big deal.)

Comment (chinandrew) [reply tests/database_metrics/test_parsers.py:1]: Should be updated now

File: tests/database_metrics/test_parsers.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+    def test_parse_db_size(self):
+        test_query_result = b'db\tsize_mb\nepidata\t7.79687500\ninformation_schema\t0.18750000\n'
+        self.assertEqual(parse_db_size(test_query_result), 7.796875)
+
+    def test_metrics(self):
+        test_metrics = (b'db\tsize_mb\nepidata\t7.79687500\ninformation_schema\t0.18750000\n',
+                        1,
+                        [{"memory_stats": {"usage": 3}}, {"memory_stats": {"usage": 4}}])
+        print(parse_metrics(test_metrics))
```
Comment (krivard) [inline tests/database_metrics/test_parsers.py:18]: Probably want to remove this `print` and add an fstring as the failure message in the assert below

**Review by krivard — CHANGES_REQUESTED** [review]

File: tests/database_metrics/test_parsers.py (hunk 2 above)
Comment (chinandrew) [reply tests/database_metrics/test_parsers.py:18]: Oops that wasn't supposed to be there, thanks for catching If it fails unittest should print out the missing dicts automatically in the assert.

Removed.

File: tests/database_metrics/test_parsers.py (hunk 1 above)
Comment (dfarrow0) [reply tests/database_metrics/test_parsers.py:1]: nice! i haven't run the tests myself, but if they pass for you that's good enough for me.

driveby tip: in case you didn't know, you can add `--full` to the line that runs tests, and it'll show coverage info on the command line. helps me write tests for all cases.

**Review by krivard — APPROVED** [review]
